### PR TITLE
Support Connections in format command

### PR DESCRIPTION
**Pull request by hod-alpert**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/28223

## Description
Added support for canvasContextConnections files in **format** command.

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Review by roysagi — APPROVED** [review]
Nice.

File: demisto_sdk/commands/format/update_connection.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+            self.remove_unnecessary_keys()
+            self.set_fromVersion(from_version=self.from_version)
+            self.save_json_to_destination_file()
+            return SUCCESS_RETURN_CODE
+        except Exception:
+            return ERROR_RETURN_CODE
+
+    def format_file(self) -> Tuple[int, int]:
+        """Manager function for the indicator type JSON updater."""
+        format = self.run_format()
```
Comment (Itay4) [inline demisto_sdk/commands/format/update_connection.py:48]: lets not shadow the builtin `format`, can use `format_` or `format_result` instead

File: demisto_sdk/commands/format/update_connection.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+                         verbose=verbose)
+
+    def run_format(self) -> int:
+        try:
+            click.secho(f'\n======= Updating file: {self.source_file} =======', fg='white')
+            self.remove_unnecessary_keys()
+            self.set_fromVersion(from_version=self.from_version)
+            self.save_json_to_destination_file()
+            return SUCCESS_RETURN_CODE
+        except Exception:
```
Comment (Itay4) [inline demisto_sdk/commands/format/update_connection.py:43]: lets also log the actual exception in verbose mode

File: demisto_sdk/commands/format/update_connection.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+                                                          SKIP_RETURN_CODE,
+                                                          SUCCESS_RETURN_CODE)
+from demisto_sdk.commands.format.update_generic_json import BaseUpdateJSON
+from demisto_sdk.tests.constants_test import CONNECTION_SCHEMA_PATH
+
+
+class ConnectionJSONFormat(BaseUpdateJSON):
+    """ConnectionJSONFormat class is designed to update connections JSON file according to Demisto's convention.
+
+       Attributes:
```
Comment (Itay4) [inline demisto_sdk/commands/format/update_connection.py:16]: please add all attributes here

**Review by Itay4 — APPROVED** [review]

**Pull request merged**